composer/utils/remote_uploader.py

```diff
@@ -12,30 +12,49 @@
 import time
 import uuid
 from concurrent.futures import Future, ProcessPoolExecutor
-from typing import List
+from typing import List, Optional
 
-from composer.utils.dist import get_local_rank
+from composer.utils.dist import broadcast_object_list, get_global_rank, get_local_rank
 from composer.utils.file_helpers import (
     maybe_create_object_store_from_uri,
+    parse_uri,
+    validate_credentials,
 )
-from composer.utils.object_store.object_store import ObjectStore, ObjectStoreTransientError
+from composer.utils.object_store.mlflow_object_store import MLFLOW_DBFS_PATH_PREFIX, MLFlowObjectStore
+from composer.utils.object_store.object_store import (
+    ObjectStore,
+    ObjectStoreTransientError,
+)
+from composer.utils.object_store.uc_object_store import UCObjectStore
 from composer.utils.retrying import retry
 
 log = logging.getLogger(__name__)
 
 __all__ = ['RemoteUploader']
 
 
+def _build_dbfs_backend(path: str) -> ObjectStore:
```

Comment:
Add this logic to `maybe_create_object_store_from_uri`?

Reply:
this is not needed in the latest commit.